Dockerfile

```diff
@@ -44,7 +44,7 @@ COPY tests/ /opt/pathml/tests
 
 # install pathml and deepcell
 RUN pip3 install --upgrade pip \
-    && pip3 install numpy==1.19.5 \
+    && pip3 install numpy==1.19.5 spams \
```

Comment:
should dockerfile have spams?

Reply:
Yes, so we can support vahadane stain normalization in the container